docs/releases/7.0.md

```diff
@@ -1,4 +1,4 @@
-# Wagtail 6.5 release notes - IN DEVELOPMENT
+# Wagtail 7.0 release notes - IN DEVELOPMENT
```

Comment:
Nitpick but when updating this file can we also align the spelling of the upgrade considerations title.

I was about to push this change but probably best to do as part of this clean up PR.

```diff
-## Upgrade considerations - changes affecting Wagtail customisations
+## Upgrade considerations - changes affecting Wagtail customizations
```